app.py

```diff
@@ -1,42 +1,60 @@
+import os
+
 import streamlit as st
+from dotenv import load_dotenv
+from openai import OpenAI
 
 def init_page():
-    '''stleamlit 初期化'''
+    """ ページ設定 """
     st.set_page_config(
-        page_title = 'Echo Bot'
+        page_title = "Echo Bot"
     )
-    st.title('Echo Bot')
+    st.title("Echo Bot")
+    if "messages" not in st.session_state:
+        st.session_state.messages = []
```

Comment:
- 変数名をchat_logにしたいです

```suggestion
    if "chat_log" not in st.session_state:
        st.session_state.chat_log = []
```